### PR TITLE
Add vscode and other editors

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: ide-setup.md [hunk 1]
```diff
@@ -51,7 +51,19 @@ To install the plugins:
...
 
-## Using the IDEs for Flutter
+### Using the IDEs for Flutter
 
 See our [additional documentation](/using-ide/) for tips on developing Flutter apps with
 Android Studio or IntelliJ IDEA.
+
+## Community supported IDEs
+
+There are number of additional IDE & editor options available enabled by community contributions:
```
Comment (sethladd) [inline ide-setup.md]: nit: break at 80 chars

File: ide-setup.md [hunk 2]
```diff
@@ -51,7 +51,19 @@ To install the plugins:
    directory, without the `bin` subdirectory. For example, `c:\Users\obiwan\flutter` / `/Users/obiwan/flutter`.
 1. Click **OK**
 
-## Using the IDEs for Flutter
+### Using the IDEs for Flutter
```
Comment (sethladd) [inline ide-setup.md]: "Using IDEs and editors for Flutter"

File: ide-setup.md [hunk 3]
```diff
@@ -51,7 +51,19 @@ To install the plugins:
...
 
 See our [additional documentation](/using-ide/) for tips on developing Flutter apps with
 Android Studio or IntelliJ IDEA.
+
+## Community supported IDEs
+
+There are number of additional IDE & editor options available enabled by community contributions:
+
+* [VS Code](https://code.visualstudio.com/) with the [Dart Code plugin](https://marketplace.visualstudio.com/items?itemName=DanTup.dart-code)
```
Comment (sethladd) [inline ide-setup.md]: aside: we might want to rebrand the Dart Code plugin to Flutter Code ?

Comment (devoncarew) [inline ide-setup.md]: Perhaps:

`with the [Dart Code](https://marketplace.visualstudio.com/items?itemName=DanTup.dart-code) plugin`

Comment (DanTup) [reply ide-setup.md]: Wouldn't it be a bit confusing to call name it after Flutter to people wanting to use Dart for non-flutter things (cli apps, web sites)?

Comment (sethladd) [reply ide-setup.md]: That's fair. Also, it's called Dart Code now so that's what we'll use here :)

Comment (sethladd) [reply ide-setup.md]: "plugin for VS Code" (for SEO)

File: ide-setup.md (hunk 2 above)
Comment (mit-mit) [reply ide-setup.md]: The page linked to here is currently IDEs only

**Pull request merged**